### DISCUSSION TITLE
Chia GUI crashed and unable to start sync on Mac

### DISCUSSION THREAD

**Discussion by Lajamieyu** [Chia-Network/chia-blockchain#6232] [Support]
Hi I had one completed plot that had not synced and another plot in progres, I tried to add another plot in queue and as a result crashed my Chia GUI. Not sure if it is because my computer is duo-core?

Since then, when I re-log on to Chia GUI, it doesn't start syncing to the mainnet and just remains offline. I am not sure how to re-sync from the same laptop. Please help and let me know if you need more information.

**Comment by BrandtH22** [comment]
@Lajamieyu  , if you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
